docs/core/introduction.md

```diff
@@ -1,57 +1,379 @@
+
+This article is about .NET 5, but much of the documentation for .NET 5 still has references to ".NET Core" or ".NET Framework". In addition, "Core" remains in the names [ASP.NET Core](/aspnet/core/) and [Entity Framework Core](/ef/core/).
+
+The documentation also refers to .NET Standard. The [.NET Standard](../standard/net-standard.md) is an API specification that lets you develop class libraries for multiple implementations of .NET.
+
+For more information, see [.NET architectural components](../standard/components.md).
+
+### Overloaded terms
+
+Some of the terminology for .NET can be confusing because the same word is used in different ways in different contexts. Here are a few of the more prominent instances:
+
+* **runtime**
+
+  |Context  |"runtime" meaning |
+  |---------|---------|
+  | [Common Language Runtime (CLR)](#clr)| The execution environment for a managed program. The OS is part of the runtime environment but isn't part of the .NET runtime. |
+  | [.NET runtime on the .NET download page](https://dotnet.microsoft.com/download/dotnet-core) | The [CLR](#clr) and [runtime libraries](#runtime-libraries), which together provide support for running [framework-dependent](#deployment-models) apps. The page also offers runtime choices for ASP.NET Core server apps and Windows desktop apps. |
+  | [Runtime Identifier (RID)](rid-catalog.md) | The OS platform and CPU architecture that a .NET app runs on. For example: Windows x64, Linux x64. |
+
+* **framework**
+
+  |Context  | "framework" meaning |
+  |---------|---------------------|
+  | .NET Framework | The original, Windows-only implementation of .NET. "Framework" is capitalized. |
+  | target framework | The collection of APIs that a .NET app or library relies on. Examples: .NET Core 3.1, .NET Standard 2.0 |
+  | Target Framework Moniker (TFM)  | A TFM is a standardized token format for specifying the target framework of a .NET app or library. Example: `net462` for .NET Framework 4.6.2. |
+  | framework-dependent app | An app that can only run on a machine where you've installed the runtime from the [.NET download page](https://dotnet.microsoft.com/download/dotnet-core). "Framework" in this usage is the same thing as the "runtime" that you download from the .NET download page. |
+  
+* **platform**
+
+  |Context  | "platform" meaning |
+  |---------|--------------------|
+  | cross platform | In this term, "platform" means an operating system and the hardware it runs on, such as Windows, macOS, Linux, iOS, and Android. |
+  | .NET platform | Usage varies. The reference may be to one implementation of .NET (such as .NET Framework or .NET 5) or to an overarching concept of .NET including all implementations. |
+
```

Comment:
I think that SDK could also be added to the overloaded list.

Reply:
Good point.  Contexts and meanings would be (1) Download page: a collection of app development tools, (2) Project file -- a set of MSBuild targets and tasks for a given app type. Are there others?

Reply:
That sounds great to me. I can't think of others.

Reply:
I added SDK.  Thanks for the review!